Tests/TABResourceLoaderTests/Services/SubclassedNetworkDataResourceServiceTests.swift

```diff
@@ -54,14 +74,20 @@ class SubclassedNetworkJSONResourceServiceTests: XCTestCase { //swiftlint:disabl
     let resourceHTTPHeaderFields = [commonKey: "resource"]
     mockResource = MockHTTPHeaderFieldsNetworkDataResource(url: mockURL, httpHeaderFields: resourceHTTPHeaderFields)
     testService.fetch(resource: mockResource) { _ in }
-    XCTAssertEqual(mockSession.capturedRequest!.allHTTPHeaderFields!, resourceHTTPHeaderFields)
+    let expectedHeaderFields = [commonKey: "resource", resourceNameKey: mockResourceName]
+    XCTAssertEqual(mockSession.capturedRequest!.allHTTPHeaderFields!, expectedHeaderFields)
```

Comment:
If you wanted to avoid the force unwraps here you could use optional chaining with a default value? 

`mockSession.capturedRequest?.allHTTPHeaderFields ?? [:]` should work? It'll still fail but won't crash the tests

Reply:
good point @KaneCheshire. i just modified what was already there. didn't even notice the force unwraps.